content/v1.12/getting-started/provider-azure-part-3.md

```diff
@@ -0,0 +1,821 @@
+            location: "Central US"
+            osDisk:
+              - caching: ReadWrite
+                storageAccountType: Standard_LRS
+            resourceGroupName: $$<resource_group_name>$$
+            size: Standard_B1ms
+            sourceImageReference:
+              - offer: debian-11
+                publisher: Debian
+                sku: 11-backports-gen2
+                version: latest
+            networkInterfaceIdsSelector:
+              matchControllerRef: true
+      patches:
+        - fromFieldPath: "spec.region"
+          toFieldPath: "spec.forProvider.location"
+          transforms:
+            - type: map
+              map: 
+                EU: "Sweden Central"
+                US: "Central US"
+    - name: quickstart-nic
+      base:
+        apiVersion: network.azure.upbound.io/v1beta1
+        kind: NetworkInterface
+        spec:
+          forProvider:
+            ipConfiguration:
+              - name: crossplane-quickstart-configuration
+                privateIpAddressAllocation: Dynamic
+                subnetIdSelector:
+                  matchControllerRef: true
+            location: "Central US"
+            resourceGroupName: $$<resource_group_name>$$
+      patches:
+        - fromFieldPath: "spec.region"
+          toFieldPath: "spec.forProvider.location"
+          transforms:
+            - type: map
+              map: 
+                EU: "Sweden Central"
+                US: "Central US"
+    - name: quickstart-subnet
+      base:
+        apiVersion: network.azure.upbound.io/v1beta1
+        kind: Subnet
+        spec:
+          forProvider:
+            addressPrefixes:
+              - 10.0.1.0/24
+            virtualNetworkNameSelector:
+              matchControllerRef: true
+            resourceGroupName: $$<resource_group_name>$$
+      patches:
+        - fromFieldPath: "spec.region"
+          toFieldPath: "spec.forProvider.location"
+          transforms:
+            - type: map
+              map: 
+                EU: "Sweden Central"
+                US: "Central US"
+    - name: quickstart-network
+      base:
+        apiVersion: network.azure.upbound.io/v1beta1
+        kind: VirtualNetwork
+        spec:
+          forProvider:
+            addressSpace:
+              - 10.0.0.0/16
+            location: "Sweden Central"
+            resourceGroupName: $$<resource_group_name>$$
+      patches:
+        - fromFieldPath: "spec.region"
+          toFieldPath: "spec.forProvider.location"
+          transforms:
+            - type: map
+              map: 
+                EU: "Sweden Central"
+                US: "Central US"
+EOF
+```
+{{< /editCode >}}
+
+### Install the Crossplane command-line
+To build a configuration package install the Crossplane Kubernetes command-line
+extension. 
+
+```shell
+wget "https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh"
+chmod +x install.sh
+./install.sh
+sudo mv kubectl-crossplane /usr/bin
```

Comment:
i get the following error with this command:
```
❯ sudo mv kubectl-crossplane /usr/bin

mv: fastcopy: open() failed (to): /usr/bin/kubectl-crossplane: Operation not permitted
```

the output of `./install/sh` tells me to copy to a different location (i guess it knows where my `kubectl` is located and tells me to put it in the same dir):
```
❯ ./install.sh
kubectl plugin downloaded successfully! Run the following commands to finish installing it:

sudo mv kubectl-crossplane /opt/homebrew/bin
kubectl crossplane --help

Visit https://crossplane.io to get started. 🚀
Have a nice day! 👋
```

Reply:
I was trying to make it easy to just copy+paste the whole block. Should I add a note about a custom kubectl install location or just remove the `mv` and tell the user to do whatever `install.sh` says?

Reply:
I think the most accurate thing to do here is to just have the user do what `install.sh` says - that determines at runtime the right place on their machine to `mv` the binary to, so it should always be accurate.